README.md

```diff
@@ -56,18 +59,23 @@ revision: [iteration of document]
 ## Previewing documentation before pushing
 
 If you wish to preview the site that will be generated before pushing, you can
-do so from your own machine.
+do so from your own machine. First install ruby:
 
-Install the `github-pages` gem (this includes `jekyll`):
+```bash
+$ apt-get install ruby ruby-dev
+$ gem install bundler [--user-install]
 
-```
-$ gem install github-pages
 ```
 
-Then you can host the site locally with the following command from the root
-directory of this repository:
+Then navigate to the root directory of this repository and run
 
+```bash
+$ bundle install [--path <installation path>]
 ```
+
+and then you can host the site locally as follows:
+
+```bash
 $ jekyll serve -w --baseurl '/xapi-project'
```

Comment:
* `bundle exec jekyll serve -w --baseurl '/xapi-project'`

Reply:
Thanks @gaborigloi, I think I'm going to put it as optional `[bundle exec]` because it's not always needed, it depends on how one has done the `bundle install`